client/app/scripts/superdesk-search/search.js

```diff
@@ -675,6 +610,21 @@
                 }
             };
 
+            var itemParameters = {
+                compact: {
+                    ITEM_HEIGHT: 57,
+                    ITEMS_COUNT: 25,
+                    BUFFER: 8
+                },
+                mgrid: {
+                    ITEM_HEIGHT: 239,
+                    ITEM_WIDTH: 190,
+                    ITEMS_COUNT: 27,
+                    ITEMS_ROW: 9,
+                    BUFFER: 18
+                }
+            };
```

Comment:
It might make sense to use freezed objects to make the intent (immutable objects containing config) even more explicit, but it's a very minor thing really...